test/lib/exporter_test.rb

```diff
@@ -74,10 +75,11 @@ def test_write_manifest
             securityContext: {}
             terminationGracePeriodSeconds: 30
     YAML
+
     File.stub( :open, DummyFile.new ) do
-      assert_output manifest do
-        @exporter.write_manifest
-      end
+      # assert_output is flaky
```

Comment:
flaky? how so?

Reply:
The test kept failing with this error : 
**_NoMethodError: undefined method `flush' for #<DummyFile:0x00007fc57f27c008>
  /Users/yinfangzhuang/.asdf/installs/ruby/2.6.6/lib/ruby/2.6.0/delegate.rb:349:in `block in delegating_block'_**

Reply:
Let me retry it

Reply:
This error is gone mysteriously at least for today

Reply:
Switch back to using assert_output